sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -30,7 +30,7 @@ async function main() {
   const client = AzureLoadTesting(endpoint, new DefaultAzureCredential());
 
   // Creating a load test
-  await client.path("/loadtests/{testId}", testId).patch({
+  const testCreationResult: any = await client.path("/loadtests/{testId}", testId).patch({
```

Comment:
We shouldn't use `: any`. Just using `const testCreationResult = await client.path("/loadtests/{testId}", testId).patch...` will give you the right type which will be an union type of a Succes | Default response type.

In order to get intellisense for the right type, you'll need to use the `isUnexpected` helper which you can import from the package. 

```ts
 const testCreationResult: any = await client.path("/loadtests/{testId}", testId).patch({
ontentType: "application/merge-patch+json",
    body: {
      displayName: displayName,
   },
  });

  if (isUnexpected(result)) {
    // If the body has an error. At this point you'll have intellisense for the error result.
    throw testCreationResult.body.error; 
  }

// In this section, intellisense should give you a union type of a `200Success | 201Success` which are really 2 different
// kind of responses, one would have returned content, the other won't. You need to handle both scenarios

if (result.status === "200") {
   // Handle the response with content, maybe log one of the properties
} else {
   // Handle the response with no content
}
```

Reply:
This is how you can get the `isUnexpected` helper
```ts
import AzureLoadTesting, {isUnexpected} from "@azure-rest/load-testing";
```

Reply:
Updated, thanks,